docs/introduction/en/papers.md

```diff
@@ -0,0 +1,22 @@
+---
+id: conflux_papers
+title: Papers
+custom_edit_url: https://github.com/Conflux-Chain/conflux-developer-site/edit/master/docs/introduction/en/papers.md
+keywords:
+  - conflux
+  - papers
+---
+
+The Conflux Protocol has solid researches to support 
+
+## [Conﬂux Protocol Specification (Yellow paper)](https://confluxnetwork.org/files/Conflux_Protocol_Specification_20201020.pdf)
+The [Protocol Specification](https://confluxnetwork.org/files/Conflux_Protocol_Specification_20201020.pdf) details described every aspect of Conflux implementation, including ledger structure, consensus, proof of work, incentive mechanism, and many others.
```

Comment:
> details described every aspect of Conflux implementation

serves as a reference of the Conflux protocol. It describes the ledger structure, the consensus mechanism, Proof of Work, the incentive mechanism, and many others aspects of the system.